storybookjs/storybook
# Use HtmlWebpackPlugin to import all assets (importing chunks in order)

Issue: https://github.com/storybooks/storybook/issues/1411

## What I did
- Removing manuel insertion of assets inside a index/iframe.html template.
- Use HtmlWebpackPlugin to import in correct order all chunks.

## How to test
- Have a build with multiple split points, everything should be loaded in order.

Is this testable with jest or storyshots?
Not sure how yet.. will do.

Does this need a new example in the kitchen sink apps?
No.

Does this need an update to the documentation?
No.

If your answer is yes to any of these, please make sure to include it in your PR.
